xml/html_title.xml

```diff
@@ -331,4 +331,16 @@
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:debian:debian_linux:-"/>
   </fingerprint>
+  <fingerprint pattern="^Eagle Eye Network">
+    <description>EagleEyeNetworks to manage IoT cameras unauthorized webpage</description>
+    <example>Eagle Eye Networks (Local)</example>
+    <param pos="0" name="service.product" value="Cloud Video Surveillance System"/>
+    <param pos="0" name="service.vendor" value="Eagle Eye Networks"/>
```

Comment:
also, you could possibly add a `<param_type>.device` param here too, maybe something camera related.

Reply:
addressed in https://github.com/rapid7/recog/pull/235/commits/0bd2353efda09e54e8afe564513701b481c9344c